doc: add descriptions for helm values. (#832)
longhorn/website: content/docs/1.6.0/monitoring/prometheus-and-grafana-setup.md

### Install Longhorn ServiceMonitor

#### Install Longhorn ServiceMonitor with Kubectl

> 1. Create a ServiceMonitor for Longhorn manager.

Reviewer: ```suggestion
Create a ServiceMonitor for Longhorn Manager.
```

Reviewer: There is only one step so we should not use an ordered list.

Author: Modified. Thanks

      - port: manager
    ```

#### Install Longhorn ServiceMonitor with Helm

1. Modify the YAML file `longhorn/chart/values.yaml`.

    ```yaml
    metrics:
      serviceMonitor:
        # -- Setting that allows the creation of a Prometheus ServiceMonitor resource for Longhorn Manager components.
        enabled: true
    ```

> 1. Create a ServiceMonitor for Longhorn manager with Helm

Reviewer: ```suggestion
1. Create a ServiceMonitor for Longhorn Manager using Helm.
```

Author: Modified. Thanks

    ```bash
      helm upgrade longhorn longhorn/longhorn --namespace longhorn-system -f values.yaml
    ```

> Longhorn ServiceMonitor has a label selector `app: longhorn-manager` for selecting Longhorn backend service.
> 
> Longhorn ServiceMonitor is included in the Prometheus custom resource so that the Prometheus server can discover all Longhorn manager pods and their endpoints.

Reviewer: ```suggestion
Longhorn ServiceMonitor is included in the Prometheus custom resource. This setup allows the Prometheus server to discover all Longhorn Manager pods and their respective endpoints.

You can use the label selector `app: longhorn-manager` to select the longhorn-backend service, which points to the set of Longhorn Manager pods.
```

Reviewer: I switched the sentences for better flow. I also added a description of the backend service to make the connection between the two points clearer.

Author: Modified. Thanks
